cypress/integration/cypressActions.spec.js

```diff
@@ -1,11 +1,96 @@
 /// <reference types='cypress' />
+const { generateUser } = require('../support/generateuser');
```

Comment:
add this file to your pull request because I can't check your homework without it